Makefile

```diff
@@ -3,15 +3,10 @@ PREFIX=/usr
 NAME=gceimgutils
 MANPATH=/usr/share/man
 dirs = lib man
-files = Makefile README.md LICENSE gceremoveimg setup.py requirements-dev.txt requirements.txt
+files = Makefile README.md LICENSE gceremoveimg gcelistimg setup.py requirements-dev.txt requirements.txt
 
-verSpec = $(shell rpm -q --specfile --qf '%{VERSION}' *.spec)
```

Comment:
Please change to `verSpec = $(shell rpm -q --specfile --qf '%{VERSION}|' *.spec | cut -d'|' -f1)` to preserve the consistency check

Reply:
Seems to be an issue with rpm parsing spec. With this change I get the same error:

```
error: line 42: Dependency tokens must begin with alpha-numeric, '_' or '/': BuildRequires:  %{python_module google-auth}
```

And I have the python rpm macros installed. 🤔

Reply:
Well it does work....
```
-> grep sle15_python_module_pythons python-gceimgutils.spec
%{?sle15_python_module_pythons}
-> rpm -q --specfile --qf '%{VERSION}|' *.spec | cut -d'|' -f1
0.13.0
```
Maybe a version issues with the python-rpm-macros package?

```
-> rpm -qa python-rpm-macros
python-rpm-macros-20240202.501440e-150500.199.1.noarch
-> grep 'python_module()' /usr/lib/rpm/macros.d/macros.python_all
.....
```

Reply:
Yeah, it was old version of rpm macros in Leap.

Reply:
Dropped the version check change as it works fine with up-to-date macros.